warehouse/query-core/src/main/java/datawave/query/jexl/visitors/PrintingVisitor.java

```diff
@@ -101,15 +101,20 @@
 import org.apache.commons.jexl3.parser.SimpleNode;
 import org.apache.commons.jexl3.parser.StringProvider;
 import org.apache.commons.jexl3.parser.TokenMgrException;
+import org.apache.log4j.Logger;
 
 import com.google.common.collect.Lists;
 
+import datawave.data.hash.AbstractUIDBuilder;
```

Comment:
This import can be removed.  So can the JexlFeatures import which was already there.